src/types/Health.ts

```diff
@@ -87,22 +101,67 @@ interface ThresholdStatus {
 const RATIO_NA = -1;
 
 export const ratioCheck = (availableReplicas: number, currentReplicas: number, desiredReplicas: number): Status => {
-  // No Pods returns No Health Info
-  if (desiredReplicas === 0 && currentReplicas === 0) {
-    return NA;
+  /*
+    IDLE STATE
+ */
+  // User performed the action
+  if (desiredReplicas === 0) {
+    return IDLE;
   }
-  // No available Pods when there are desired and current means a Failure
+
+  /*
+   DEGRADED STATE
```

Comment:
In similar way, a comment like this will help to refresh/understand the logic (at least for me):
```
// When a workload has available pods but less than desired defined by user it should be marked as degraded
```